Taskfile.yml

```diff
@@ -197,6 +202,62 @@ tasks:
           --name {{.MOCK_GENERATE_INTERFACE_NAME}} \
           --output {{.MOCK_GENERATE_DIR}}/mocks \
           --tags {{.BUILD_TAGS}}
+  opa-fmt:
+    desc: check formatting rego files using opa
+    silent: true
+    cmds:
+      - task: opa-install
+      - "{{.OPA_FMT}}"
+  opa-fmt-write:
+    desc: apply formatting rego files using opa
+    silent: true
+    cmds:
+      - task: opa-install
+      - "{{.OPA_FMT}} --write"
+  opa-install:
+    silent: true
+    cmds:
+      - |
+        if ! opa version | grep -q {{.OPA_VERSION}}; then
+          go install github.com/open-policy-agent/opa@{{.OPA_VERSION}}
+        fi
+  opa-run:
+    silent: true
+    cmds:
+      - |
+        for dir in $(find . -type f -name '*.rego' -exec dirname {} \; | sort -u); do
+          echo "Running opa test in directory: $dir" && (cd "$dir" && opa test . -v)
+        done
+  opa:
+    desc: run opa
+    silent: true
+    cmds:
+      - task: opa-install
+      - task: opa-run
+  regal-install:
+    silent: true
+    cmds:
+      - |
+        if ! regal version | grep -q {{.REGAL_VERSION}}; then
+          # regal version installed using `go install` does not include
+          # version, see: https://github.com/StyraInc/regal/issues/1275
+          curl -L -o {{.REGAL}} https://github.com/StyraInc/regal/releases/download/v{{.REGAL_VERSION}}/regal_Darwin_arm64
+          chmod +x {{.REGAL}}
+        fi
+  regal-run:
+    silent: true
+    cmds:
+      - |
+        if ! regal lint .; then
+          echo "regal linting failed. Run: 'task remote:opa-fmt-write' to fix the issue"
```

Comment:
Nice to have a concrete error message with instructions how to solve the issue. ❤️